examples/todoapp-lite/shared/src/commonMain/kotlin/example/todoapp/lite/common/RootContent.kt

```diff
@@ -11,7 +13,7 @@ fun RootContent(modifier: Modifier = Modifier) {
     val state = model.state
 
     MainContent(
-        modifier = modifier,
+        modifier = modifier.background(MaterialTheme.colors.background),
```

Comment:
Workaround for https://github.com/JetBrains/compose-multiplatform-core/pull/812
But I'd keep it even after this fix to align styling with material spec